src/Auth/AuthContainer.js

```diff
@@ -16,7 +16,7 @@ function AuthContainer({ children }) {
 }
 
 AuthContainer.propTypes = {
-  children: PropTypes.object,
+  children: PropTypes.shape(),
```

Comment:
Generally, if a property doesn't have `` isRequired ``, it should have default props using `` 
defaultProps ``.

This eliminates the case when a property is access but its value is undefined. See https://jsfiddle.net/69z2wepo/64812/ for demo.